src/components/new-request/search.vue

```diff
@@ -242,46 +242,44 @@
 
       <!-- Restoration / Reinstatement flow -->
       <template v-else-if="isRestoration">
-        <BusinessLookupFetch ref="MyBusinessLookup" />
+        <BusinessLookupFetch />
         <SocietiesInfo
           v-if="isSocietyDisabled && showSocietiesInfo"
           :type="'restore'"
           :showDialog="showSocietiesInfo"
         />
-        <template v-else>
```

Comment:
Go to ⚙️ button above and hide whitespace changes.

All I did here was remove this code from the "else" block. This was a logic error. I tested thoroughly -- see ticket for screenshots.